draft-vvv-webtransport-quic.md

```diff
@@ -96,7 +96,9 @@ The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
-This document follows terminology defined in Section 1.2 of [OVERVIEW].
+This document follows terminology defined in Section 1.2 of [OVERVIEW].  The
+diagrams describe encoding following the conventions described in Section 1.3 of
+[QUIC-TRANSPORT].
```

Comment:
To get this to turn into a link, I think you want `{{QUIC-TRANSPORT}}` with a definition similar to what's in draft-ietf-quic-tls.

Reply:
It's already a link, though.

Reply:
Huh, weird